examples/scene/prefab.rs

```diff
@@ -0,0 +1,205 @@
+                    }
+                    // We insert components from the first scene entity onto the provided entity.
+                    // If there are multiple scene_entites, they will be placed on newly spawned entities. 
+                    for (idx, scene_entity) in scene.entities.iter().enumerate() {
+                        let new_entity = match idx == 0 {
+                            true => entity,
+                            false => world.spawn().id(),
+                        };
+                        for component in scene_entity.components.iter() {
+                            // Remember to register any components you want spawned!
+                            let registration = type_registry
+                                .get_with_name(component.type_name())
+                                .ok_or_else(|| SceneSpawnError::UnregisteredType {
+                                    type_name: component.type_name().to_string(),
+                                }).unwrap();
+                            let reflect_component =
+                                registration.data::<ReflectComponent>().ok_or_else(|| {
+                                    SceneSpawnError::UnregisteredComponent {
+                                        type_name: component.type_name().to_string(),
+                                    }
+                                }).unwrap();
+                            if world
+                                .entity(new_entity)
+                                .contains_type_id(registration.type_id())
+                            {
+                                if entry.overwrite_existing_components {
+                                    reflect_component.apply_component(world, new_entity, &**component);
+                                }
+                            } else {
+                                reflect_component.add_component(world, new_entity, &**component);
+                            }
+                        }
+                    }
+
+                    complete.push(entity);
+                }
+            };
+        });
+        
+        // Remove the entity from the queue.
+        for entity in complete {
+            factory.queue.remove(&entity);
+        }
+
+    })
+}
+
+fn write_scene(world: &mut World) {
+    let mut scene_world = World::new();
+    
+    let path = "assets/my_scene/path.scn.ron";
+
+    scene_world
+        .spawn()
+        .insert(A {
+            message: "I came from the scene file!".to_string()
+        })
+        .insert(B {
+            data: vec![
+                5, 7, 8, 22, 42, 1001
+            ]
+        })
+    ;
+    
+    let type_registry = world.get_resource::<TypeRegistry>().unwrap();
+    let scene = DynamicScene::from_world(&scene_world, type_registry);
+
+    let mut file = match File::create(path) {
+        Err(why) => panic!("Failed to create file: {}", why),
```

Comment:
```suggestion
        Err(reason) => panic!("Failed to create file: {}", reason),
```
More consistent; slightly clearer to non-native speakers.